src/main/java/org/spongepowered/api/boss/ServerBossBar.java

```diff
@@ -0,0 +1,123 @@
+package org.spongepowered.api.boss;
+
+import org.spongepowered.api.Server;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.util.ResettableBuilder;
+
+import java.util.Collection;
+
+/**
+ * Represents a boss bar controlled by a {@link Server}.
+ */
+public interface ServerBossBar extends BossBar {
+
+    /**
+     * Gets a collection of all players on this boss bar.
+     *
+     * @return A collection of all players on this boss bar
+     */
+    Collection<Player> getPlayers();
+
+    /**
+     * Adds a player to this boss bar.
+     *
+     * @param player The player to add
+     */
+    void addPlayer(Player player);
+
+    /**
+     * Removes a player from this boss bar.
+     *
+     * @param player The player to remove
+     */
+    void removePlayer(Player player);
+
+    /**
+     * If this boss bar is visible.
+     *
+     * @return If this boss bar is visible
+     */
+    boolean isVisible();
```

Comment:
I think what @Zidane is trying to say is why is this only in `ServerBossBar` and not in the superclass? There's no way that you can't hide the boss bar on the client.